cob_hardware_config/cob4-1/config/teleop.yaml

```diff
@@ -1,20 +1,97 @@
+#cob4-1
+##########
 # common params
 run_factor: 2.0
+joy_num_buttons: 12
+joy_num_axes: 6
+joy_num_modes: 4
+mode_switch_button: 8
+light_topic_name: /light_torso/light
+
+# axes
+axis_vx: 1
+axis_vy: 0
+axis_vz: 5
+axis_roll: 4
+axis_pitch: 3
+axis_yaw: 2
 
 # buttons
 deadman_button: 5
+safety_button: 4
+init_button: 9
+
+#mode1: Base
 run_button: 7
-base_safety_button: 4
 
-# axes
-axis_vx: 1
-axis_vy: 0
-axis_vth: 2
+#mode2: Trajectory controller
+torso_sss_default_target_button: 1
+sensorring_sss_default_target_button: 3
+head_sss_default_target_button: 3
```

Comment:
is it ok to use the same button for two components? both head and sensorring on `3`?